jetty-jaas/pom.xml

```diff
@@ -166,6 +166,28 @@
         </pluginManagement>
       </build>
     </profile>
+    <profile>
+      <id>over-jdk17</id>
+      <activation>
+        <jdk>[21,)</jdk>
```

Comment:
Should be `[18,)`, also given the `id`.

Reply:
Good point about the id.
But this isn't necessary on JDK 18 or JDK 19 (both can run the test case fine).
I'll correct the id.

Reply:
Pushed another commit.

Tested the following with this change ...

* JDK11 - JAASLdapLoginServiceTest is tested and works
* JDK17 - JAASLdapLoginServiceTest is tested and works
* JDK18 - JAASLdapLoginServiceTest is tested and works
* JDK19 - JAASLdapLoginServiceTest is tested and works
* JDK20 - JAASLdapLoginServiceTest does not work, and is now skipped
* JDK21 - JAASLdapLoginServiceTest does not work, and is now skipped